EvenOddNumbers.cls

```diff
@@ -0,0 +1,13 @@
+public class EvenOddNumbers {
+    public static void checkNumber(Integer num){

```

Comment:
@pranalimm  Logic is right but as method name is `checkNumber` it's pretend that this method will only checking the number which is passed as param is Even or Odd so according to me this method should return Boolean then you can print based on return value.

for example,

```java
/*
   This method will return true is it's even number else will return false.
*/
public static boolean checkEvenOrOdd(Integer num){
     boolean isEven = true;
     if(math.mod(num,2) != 0){
          isEven = false;
     }
     return isEven;
}
```